model/slide/DBSlide.ts

```diff
@@ -0,0 +1,34 @@
+import { BaseEntity, Entity, Column, PrimaryGeneratedColumn } from 'typeorm';
+import { ISlide } from './ISlide';
+import { Content } from './content/Content';
+import { LayoutSlot, LayoutType } from './layout/Layout';
+
+/**
+ * Database object of a slide.
+ */
+@Entity()
+export class DBSlide extends BaseEntity implements ISlide {
+    /** Layout of this slide */
+    @Column()
+    layout: LayoutType;
+
+    /** Content of each slot from the layout of the slide */
+    @Column('simple-json')
+    content: Record<LayoutSlot, Content>;
+
+    /** Background color of this slide */
+    @Column()
+    backgroundColor: string;
+
+    /** Code of the sequence to which this slides belongs */
+    @Column()
+    sequenceCode: string;
```

Comment:
might be worth indexing slides by the sequence code, might speed up querying for them